README.md

```diff
@@ -150,6 +152,7 @@ end
 Get replication lag
 
 ```ruby
+# returns the replication_lag in seconds
```

Comment:
Let's put this above (outside the code block)

Reply:
Done